lib/aur-sync

```diff
@@ -108,58 +80,55 @@ if [[ -t 2 && ! -o xtrace ]]; then
     colorize
 fi
 
-opt_short='d:D:AcfgklLpstTu'
-opt_long=('bind:' 'bind-rw:' 'database:' 'directory:' 'ignore:' 'repo:' 'root:'
-          'makepkg-conf:' 'pacman-conf:' 'allan' 'chroot' 'continue'
-          'force' 'ignore-arch' 'ignorearch' 'log' 'noconfirm' 'no-confirm'
-          'no-ver' 'nover' 'nover-shallow' 'noprovides' 'no-provides'
-          'no-ver-shallow' 'no-view' 'noview' 'print' 'rm-deps'
-          'keep' 'sign' 'temp' 'tar' 'upgrades' 'list' 'git'
-          'repo-list' 'rebuild' 'rebuildtree' 'rebuild-tree')
+opt_short='d:D:AcfgkLPstTuh'
+opt_long=('bind:' 'bind-rw:' 'database:' 'directory:' 'ignore:'
+          'repo:' 'root:' 'makepkg-conf:' 'pacman-conf:' 'allan'
+          'chroot' 'continue' 'force' 'ignore-arch' 'ignorearch' 'log'
+          'noconfirm' 'no-confirm' 'no-ver' 'nover' 'nover-shallow'
+          'no-ver-shallow' 'no-view' 'noview' 'print' 'provides'
+          'rm-deps' 'keep' 'sign' 'temp' 'tar' 'upgrades' 'git'
+          'rebuild' 'rebuildtree' 'rebuild-tree' 'help')
 
 if ! parseopts "$opt_short" "${opt_long[@]}" -- "$@"; then
     usage
 fi
 set -- "${OPTRET[@]}"
 
-unset pkg pkg_i root repo
+unset pkg pkg_i
 while true; do
     case "$1" in
-        -d|--repo|--database) shift; repo=$1 ;;
+        -d|--repo|--database) shift; repo_args+=(-d "$1") ;;
```

Comment:
`aur-sync` is the only binary accepting `--repo` as an alternative to `-d` or `--database`. Shouldn't we had this to the other binaries for consistency?

Reply:
I agree it should be consistent. I'm unsure if `--repo` is such a useful alias though.